### PR TITLE
New screenshots and related text for new Compass tab arrangement and button la…

### PR DESCRIPTION
**Pull request by rob-mongodb**
…bels: DOCSP-748

Staged at https://docs-mongodbcom-staging.corp.mongodb.com/compass/robertjustice/DOCSP-748/query-bar.html.

### CODE REVIEW

File: source/query-bar.txt [hunk 1]
```diff
@@ -57,7 +57,7 @@ operator.
    Builder to enter a query into the query bar. For details, see
    :ref:`build-query`.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
```
Comment (atsansone) [inline source/query-bar.txt]: I would break the sentence at "disabled" and start with "The Filter label appears red until..." ("turns" implies that it started as black then became red.)

File: source/query-bar.txt [hunk 2]
```diff
@@ -57,7 +57,7 @@ operator.
...
 :guilabel:`Filter` label turns red until a valid query is entered.
 Filter syntax requires strict :manual:`extended JSON
 </reference/mongodb-extended-json/>` types. For instance, to specify an
```
Comment (atsansone) [inline source/query-bar.txt]: Why not show this as an `.. example::`? It would call it out better.

File: source/query-bar.txt [hunk 3]
```diff
@@ -69,7 +69,7 @@ as in the following filter example:
    { "_id": { "$oid": "58eee4f43d1a10a4e5339aac" } }
 
 When you have entered a valid MongoDB query, the :guilabel:`Filter`
```
Comment (atsansone) [inline source/query-bar.txt]: This paragraph duplicates the one starting on line 60. I don't know if the Filter issue needs to be called out twice.

File: source/query-bar.txt [hunk 4]
```diff
@@ -69,7 +69,7 @@ as in the following filter example:
...
-label is no longer red and :guilabel:`Apply` button is enabled. 
+label is no longer red and :guilabel:`Find` button is enabled.
```
Comment (atsansone) [inline source/query-bar.txt]: We usually don't specify the UI element unless it is unclear which element you mean (i.e. two elements named "Find": one link and one button).

File: source/query-bar.txt [hunk 5]
```diff
@@ -146,7 +146,7 @@ maximum number of documents to return. To specify the limit, click
 Run the Query
 ~~~~~~~~~~~~~
 
-Click the :guilabel:`Apply` button to run the query and view the
+Click the :guilabel:`Find` button to run the query and view the
```
Comment (atsansone) [inline source/query-bar.txt]: See [line 72](https://github.com/mongodb/docs-compass/pull/12/files#diff-93adb5021a0a047de98a19b6c0073135R72).

File: source/query-plan.txt [hunk 6]
```diff
@@ -16,7 +16,7 @@ query.
 To view the execution plan for a query filter, click on the collection
 on the left hand pane and select the :guilabel:`Explain Plan` tab.
```
Comment (atsansone) [inline source/query-plan.txt]: "Click" tab, not "Select".

File: source/query-bar.txt [hunk 7]
```diff
@@ -69,7 +69,7 @@ as in the following filter example:
    { "_id": { "$oid": "58eee4f43d1a10a4e5339aac" } }
 
 When you have entered a valid MongoDB query, the :guilabel:`Filter`
-label is no longer red and :guilabel:`Apply` button is enabled. 
+label is no longer red and :guilabel:`Find` button is enabled. 
 
 Project
 ~~~~~~~
```
Comment (atsansone) [inline source/query-bar.txt]: Note how you wrote [Line 85](https://github.com/mongodb/docs-compass/pull/12/files#diff-93adb5021a0a047de98a19b6c0073135R85) vs. [Lines 94-95]https://github.com/mongodb/docs-compass/pull/12/files#diff-93adb5021a0a047de98a19b6c0073135R94-95). I think that the latter is better, i.e. just the colon and no "for example".

File: source/includes/extracts-query-bar.yaml [hunk 8]
```diff
@@ -4,14 +4,40 @@ content: |
...
+ref: query-bar-type-schema
 content: |
    You can type MongoDB queries into the query bar. In the query
    expression, you can use the various MongoDB :manual:`query operators
    </reference/operator/query/>` except for the :query:`$text` query
    operator.
 
-   As you type, the :guilabel:`Apply` button is disabled and the
+   As you type, the :guilabel:`Analyze` button is disabled and the
+   :guilabel:`Filter` label turns red until a valid query is entered.
```
Comment (atsansone) [inline source/includes/extracts-query-bar.yaml]: I would break the sentence at "disabled" and start with "The Filter label appears red until..." ("turns" implies that it started as black then became red.)

File: source/includes/extracts-query-bar.yaml [hunk 9]
```diff
@@ -4,14 +4,40 @@ content: |
...
+---
+ref: query-bar-type-documents
+content: |
+   You can type MongoDB queries into the query bar. In the query
+   expression, you can use the various MongoDB :manual:`query operators
+   </reference/operator/query/>` except for the :query:`$text` query
+   operator.
+
+   As you type, the :guilabel:`Find` button is disabled and the
+   :guilabel:`Filter` label turns red until a valid query is entered.
```
Comment (atsansone) [inline source/includes/extracts-query-bar.yaml]: See [Line 15](https://github.com/mongodb/docs-compass/pull/12/files#diff-f29e92b96c7166c8129d1c34b89b8404R15).

File: source/includes/extracts-query-bar.yaml [hunk 10]
```diff
@@ -4,14 +4,40 @@ content: |
...
+---
+ref: query-bar-type-queryplan
+content: |
+   You can type MongoDB queries into the query bar. In the query
+   expression, you can use the various MongoDB :manual:`query operators
+   </reference/operator/query/>` except for the :query:`$text` query
+   operator.
+
+   As you type, the :guilabel:`Explain` button is disabled and the
    :guilabel:`Filter` label turns red until a valid query is entered.
```
Comment (atsansone) [inline source/includes/extracts-query-bar.yaml]: See [Line 15](https://github.com/mongodb/docs-compass/pull/12/files#diff-f29e92b96c7166c8129d1c34b89b8404R15).

Comment (atsansone) [inline source/includes/extracts-query-bar.yaml]: See [Line 15](https://github.com/mongodb/docs-compass/pull/12/files#diff-f29e92b96c7166c8129d1c34b89b8404R15).

File: source/includes/steps-create-index.yaml [hunk 11]
```diff
@@ -11,7 +11,7 @@ level: 4
 ref: enter-index-name
 content: |
    In the dialog, enter the name of the index to create, or leave blank
-   to have MongODB create a default name for the index.
+   to have MongoDB create a default name for the index.
```
Comment (atsansone) [inline source/includes/steps-create-index.yaml]: "...create an index with a default name." What is a default name?

File: source/includes/steps-create-query-via-builder.yaml [hunk 12]
```diff
@@ -45,6 +45,6 @@ title: Run the query
 level: 4
 ref: run-query
 content: |
-   To run the query, click the :guilabel:`Apply` button. You can clear
+   To run the query, click the :guilabel:`Analyze` button. You can clear
```
Comment (atsansone) [inline source/includes/steps-create-query-via-builder.yaml]: We usually don't specify the UI element unless it is unclear which element you mean (i.e. two elements named "Find": one link and one button).

File: source/includes/steps-create-query-via-builder.yaml [hunk 13]
```diff
@@ -45,6 +45,6 @@ title: Run the query
...
    your query by pressing the :guilabel:`Reset` button.
```
Comment (atsansone) [inline source/includes/steps-create-query-via-builder.yaml]: "Click Reset to clear the query."

File: source/query-bar.txt (hunk 1 above)
Comment (atsansone) [inline source/query-bar.txt]: We usually don't specify the UI element unless it is unclear which element you mean (i.e. two elements named "Find": one link and one button).

File: source/query-bar.txt (hunk 2 above)
Comment (atsansone) [inline source/query-bar.txt]: Or, you could just have it say: "To specify an ``ObjectID``, use the format ``{ "$oid": "<string>" }``:"

File: source/documents.txt [hunk 14]
```diff
@@ -45,7 +45,7 @@ tab loads, Compass displays all documents that match your query criteria
 (or all documents within the sample if no query criteria was
 specified).
 
-.. figure:: /images/compass/doc-view.svg
+.. figure:: /images/compass/doc-view.png
   :figwidth: 800px
```
Comment (atsansone) [inline source/documents.txt]: This will stretch the image in an odd fashion. You should keep it to its exact size: `810px`. I would check the rest accordingly.

File: source/query-bar.txt (hunk 1 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: I believe that is the reason the text is using the full element name: to distinguish between the Find button, which is disabled, and the Filter label, which does in fact turn red after starting out black.  Also note that these two actions simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential  value.

File: source/query-bar.txt (hunk 2 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Replaced code-block with ..example:: tag.

File: source/query-bar.txt (hunk 3 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Removed.

File: source/query-bar.txt (hunk 4 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: See previous comment. Only using the full element name because this feature is a little tricky the first time that you use it.

File: source/query-bar.txt (hunk 7 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Fixed.  Also, I replaced  code-block with example blocks, per your earlier comment.

File: source/query-bar.txt (hunk 5 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Removed "button".

File: source/query-plan.txt (hunk 6 above)
Comment (rob-mongodb) [inline source/query-plan.txt]: Fixed. Also removed "tab".

File: source/includes/extracts-query-bar.yaml (hunk 8 above)
Comment (rob-mongodb) [inline source/includes/extracts-query-bar.yaml]: Yup.  It really does change color. See earlier response.

File: source/query-bar.txt (hunk 5 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Note that these two actions simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential value.

File: source/includes/extracts-query-bar.yaml (hunk 9 above)
Comment (rob-mongodb) [inline source/includes/extracts-query-bar.yaml]: Excerpt from earlier response: "Note that these two actions occur simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential value."

File: source/includes/extracts-query-bar.yaml (hunk 10 above)
Comment (rob-mongodb) [inline source/includes/extracts-query-bar.yaml]: Excerpt from earlier response: "Note that these two actions occur simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential value."

Comment (rob-mongodb) [inline source/includes/extracts-query-bar.yaml]: Excerpt from earlier response: "Note that these two actions occur simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential value."

File: source/includes/steps-create-index.yaml (hunk 11 above)
Comment (rob-mongodb) [inline source/includes/steps-create-index.yaml]: "Default" isn't a technical term, and I believe it's common knowledge as a concept. Not worth a glossary entry or a dictionary definition, I think.

File: source/includes/steps-create-query-via-builder.yaml (hunk 12 above)
Comment (rob-mongodb) [inline source/includes/steps-create-query-via-builder.yaml]: Fixed.

File: source/includes/steps-create-query-via-builder.yaml (hunk 13 above)
Comment (rob-mongodb) [inline source/includes/steps-create-query-via-builder.yaml]: Fixed.

File: source/documents.txt (hunk 14 above)
Comment (rob-mongodb) [inline source/documents.txt]: Good catch. Fixed.

**Comment by rob-mongodb** [comment]
@atsansone , ready for another look.  Staging is updated.

File: source/schema.txt [hunk 15]
```diff
@@ -39,7 +39,7 @@ specify query options.
 .. figure:: /images/compass/query-bar-schema-view.png
    :figwidth: 730px
 
-.. include:: /includes/extracts/query-bar-type.rst
+.. include:: /includes/extracts/query-bar-type-schema.rst
 
 .. tip::
```
Comment (atsansone) [inline source/schema.txt]: You could just link *Query Builder* and skip the "For details..."

File: source/query-bar.txt [hunk 16]
```diff
@@ -146,8 +143,7 @@ maximum number of documents to return. To specify the limit, click
 Run the Query
 ~~~~~~~~~~~~~
 
-Click the :guilabel:`Apply` button to run the query and view the
-updated results.
+Click :guilabel:`Find` to run the query and view the updated results.
 
 .. note::
```
Comment (atsansone) [inline source/query-bar.txt]: You can change "click the Reset button" to just "click Reset". There is nothing else with which that could be confused.

File: source/query-bar.txt [hunk 17]
```diff
@@ -8,7 +8,7 @@ From the various Collections tabs, you can use the query bar to specify
 MongoDB queries.  For details, see :ref:`querying`.
 
 .. figure:: /images/compass/query-bar-documents-view.png
-   :figwidth: 780px
+   :figwidth: 805px
 
 You can specify a query filter in the query bar. If you click the
```
Comment (atsansone) [inline source/query-bar.txt]: In the graf below, why not link to project, sort, skip and limit?

File: source/query-bar.txt [hunk 18]
```diff
@@ -8,7 +8,7 @@ From the various Collections tabs, you can use the query bar to specify
 MongoDB queries.  For details, see :ref:`querying`.
 
 .. figure:: /images/compass/query-bar-documents-view.png
-   :figwidth: 780px
+   :figwidth: 805px
 
 You can specify a query filter in the query bar. If you click the 
 :guilabel:`Options` to expand the query bar, you also have access to
```
Comment (atsansone) [inline source/query-bar.txt]: I would shorten this to "...you can use..."

File: source/includes/extracts-query-bar.yaml [hunk 19]
```diff
@@ -1,17 +1,43 @@
 ref: query-bar-results
 content: |  
-   For large query result sets, the results are shown from a sampled       set of documents.  Otherwise, Compass shows the entire result set.
+   For large query result sets, the results are shown from a sampled set of documents.  Otherwise, Compass shows the entire result set.
```
Comment (rob-mongodb) [inline source/includes/extracts-query-bar.yaml]: Fixed.

File: source/query-bar.txt (hunk 17 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Good suggestion.  Added links to aggregation operators.

Previous two comments do not have a reply button, so including responses here:
* inclusive/exclusive: I think you're confusing 'limit' with "range".  This is a single integer, not a series/range, so the answer is implicitly inclusive.  Note that we don't bother to point this out even on the reference page that I've now linked to.
* define default: I'm going to invoke writer's privilege here and move on.

File: source/query-bar.txt (hunk 18 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: I like 'specify', and the four-letter difference in word length is negligible.

File: source/query-bar.txt [hunk 20]
```diff
@@ -57,20 +57,17 @@ operator.
    Builder to enter a query into the query bar. For details, see
    :ref:`build-query`.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
 :guilabel:`Filter` label turns red until a valid query is entered.
 Filter syntax requires strict :manual:`extended JSON
-</reference/mongodb-extended-json/>` types. For instance, to specify an
+</reference/mongodb-extended-json/>` types. To specify an
 ``ObjectID``, use the format ``{ "$oid": "<string>" }``
```
Comment (rob-mongodb) [inline source/query-bar.txt]: I don't think the text disagrees with that assertion. Not sure what your point is, but I added a link to ObjecttId.

File: source/schema.txt (hunk 15 above)
Comment (rob-mongodb) [inline source/schema.txt]: Made the change here and also in query-bar.txt.

File: source/query-bar.txt (hunk 16 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Fixed.

File: source/query-bar.txt [hunk 21]
```diff
@@ -57,20 +57,17 @@ operator.
...
 as in the following filter example:
 
-.. code-block:: javascript
+.. example::
 
    { "_id": { "$oid": "58eee4f43d1a10a4e5339aac" } }
```
Comment (rob-mongodb) [inline source/query-bar.txt]: Fixed.

File: source/query-bar.txt [hunk 22]
```diff
@@ -82,9 +79,9 @@ To set projection, click :guilabel:`Options` and enter in the
 projection document:
 
 - To include fields, specify the field name and set to ``1`` in the
-  project document. For example:
+  project document:
 
-  .. code-block:: javascript
+  .. example::
```
Comment (rob-mongodb) [inline source/query-bar.txt]: Fixed.

File: source/query-bar.txt [hunk 23]
```diff
@@ -94,7 +91,7 @@ projection document:
 - To exclude fields, specify the field name and set to ``0`` in the
   project document:
 
-  .. code-block:: javascript
+  .. example::
 
      { year: 0, name: 0 }
```
Comment (rob-mongodb) [inline source/query-bar.txt]: Fixed.

File: source/query-bar.txt [hunk 24]
```diff
@@ -124,7 +121,7 @@ in the sort documents. For example, the following sort document sorts
 results first by ``year`` in descending order, and within each year, 
 sort by ``name`` in ascending order.
 
-.. code-block:: javascript
+.. example::
 
    { year: -1, name: 1 }
```
Comment (rob-mongodb) [inline source/query-bar.txt]: Fixed.

**Comment by rob-mongodb** [comment]
@atsansone , ready for another look. Staging updated. 

Note: the only places where "query builder" are capitalized are in headers for heading capitalization and in links to the reference doc (:ref:`build-query).

**Comment by atsansone** [comment]
Only nit: You can nest `example`  and `code-block`:

`.. example::`
`    .. code-block:: json`

File: source/query-bar.txt [hunk 25]
```diff
@@ -8,36 +8,36 @@ From the various Collections tabs, you can use the query bar to specify
...
 .. list-table::
    :widths: 15 85
 
-   * - ``Filter``
+   * - Filter
 
      - Filter for the schema view.
 	 
-   * - ``Project``
+   * - `Project <https://docs.mongodb.com/manual/reference/operator/aggregation/project/>`_
```
Comment (kay-kim) [inline source/query-bar.txt]: these are the wrong links. 
Please review the following sections in the manual:

https://docs.mongodb.com/manual/crud/
https://docs.mongodb.com/manual/core/aggregation-pipeline/

File: source/query-bar.txt [hunk 26]
```diff
@@ -82,8 +79,9 @@ To set projection, click :guilabel:`Options` and enter in the
 projection document:
 
 - To include fields, specify the field name and set to ``1`` in the
-  project document. For example:
+  project document:
 
+  .. example::
```
Comment (kay-kim) [inline source/query-bar.txt]: This and others like it do not render correctly.

File: source/query-bar.txt (hunk 25 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: @kay-kim , I don't understand the comment. I'm trying to link to the references pages for these aggregation operators, just as the text does in the aggregation-pipeline doc that you refer to.  Will follow up with you tomorrow.

File: source/query-bar.txt (hunk 26 above)
Comment (rob-mongodb) [inline source/query-bar.txt]: Fixed.  Also added " ... as in following example" text in place of the Example directive.

File: source/query-bar.txt (hunk 25 above)
Comment (kay-kim) [inline source/query-bar.txt]: @rob-mongodb  -- So, I wanted you to review the sections:
1) to see that aggregation and find operations are two separate things.   I believe these are project/sort/limit ... associated with find operations.
2) to ensure that you are familiar with the two (not in the abstract but in the concrete sense) -- so that you know how to use them, that you recognize they are separate things, etc.

Comment (rob-mongodb) [inline source/query-bar.txt]: @kay-kim , links updated at https://docs-mongodbcom-staging.corp.mongodb.com/compass/robertjustice/DOCSP-748/query-bar.html. Ready for another look.

File: source/query-bar.txt [hunk 27]
```diff
@@ -53,24 +53,20 @@ operator.
...
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
 :guilabel:`Filter` label turns red until a valid query is entered.
 Filter syntax requires strict :manual:`extended JSON
-</reference/mongodb-extended-json/>` types. For instance, to specify an
-``ObjectID``, use the format ``{ "$oid": "<string>" }``
-as in the following filter example:
+</reference/mongodb-extended-json/>` types. To specify an
+`ObjectID <https://docs.mongodb.com/manual/reference/method/ObjectId/>`_,
```
Comment (kay-kim) [inline source/query-bar.txt]: For future reference -- if you look at line 61-62, that's another way to link to the manual.  Don't bother updating this commit.   More for you to know in future.

**Comment by kay-kim** [comment]
merged!
